### PR TITLE
feat: 🌟 Support Crop and tf>2.x.x for the UMSI Metric

### PR DESCRIPTION
**Pull request by kargaranamir**

### CODE REVIEW

File: backend/aim/metrics/m9/m9_umsi.py [hunk 1]
```diff
@@ -419,7 +461,7 @@ def execute_metric(
             "aim/metrics/m9/xception_cl_fus_aspp_imp1k_10kl-3cc0.1mse-5nss5bc_bs4_ep30_valloss-2.5774_full.h5"
         )
         model: keras.engine.training.Model = keras.models.load_model(
-            model_filepath
+            model_filepath, compile=False, custom_objects={"K": K}
```
Comment (kargaranamir) [inline backend/aim/metrics/m9/m9_umsi.py:464]: We handled the issue with Tensorflow 2.x.x HERE.

File: backend/aim/metrics/m9/m9_umsi.py [hunk 2]
```diff
@@ -441,15 +483,21 @@ def execute_metric(
                 original_images, heatmap_batch, heatmap_overlay_batch
             )
 
+        # Concatenate all batch cropped images
```
Comment (kargaranamir) [inline backend/aim/metrics/m9/m9_umsi.py:486]: We have concatenated the umsi output of all cropped images here. Since the code itself is designed to work with multiple images, we use the multiple image feature. When you are using a GPU, this feature allows for more speed.
